Make error

Errors in make process 
cmake version 3.6.2;
Ubuntu 13.10

```
[ 92%] Building CXX object src/artm_tests/CMakeFiles/artm_tests.dir/thread_safe_holder_test.cc.o
/home/oppex/bigartm-test/bigartm/src/artm_tests/thread_safe_holder_test.cc: In member function 'virtual void Async_Boost_Test::TestBody()':
/home/oppex/bigartm-test/bigartm/src/artm_tests/thread_safe_holder_test.cc:40:96: error: conversion from 'boost::unique_future<void>' to non-scalar type 'boost::unique_future<int>' requested
   boost::unique_future<int> fut = boost::async(boost::launch::async, [input](){ return input; });
                                                                                                ^
make[2]: *** [src/artm_tests/CMakeFiles/artm_tests.dir/thread_safe_holder_test.cc.o] Error 1
make[1]: *** [src/artm_tests/CMakeFiles/artm_tests.dir/all] Error 2
make: *** [all] Error 2
```

--- a/src/artm_tests/thread_safe_holder_test.cc
+++ b/src/artm_tests/thread_safe_holder_test.cc
@@ -35,13 +35,6 @@ TEST(ThreadSafeHolder, Basic) {
 
 // To run this particular test:
 // artm_tests.exe --gtest_filter=Async.*
-TEST(Async, Boost) {
-  int input = 123;
-  boost::unique_future<int> fut = boost::async(boost::launch::async, [input](){ return input; });
-  int output = fut.get();
-  ASSERT_EQ(input, output);
-}
-
 TEST(Async, Std) {
   int input = 123;
   std::future<int> fut = std::async(std::launch::async, [input](){ return input; });

